From 51b7928a55155dc2a462cef5d4c045a81d9c1e0c Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Mon, 3 Sep 2018 23:32:09 +0200
Subject: [PATCH] modules: update OpenWrt packages

40da7ecf21ff socat: Fix CRDLY, TABDLY and CSIZE shifts for PowerPC
a5bbf27e3532 libxml2: add Debian patches to address CVEs
1e77dfa7b084 tiff: fix remaining CVEs
cad5ceed6a87 unbound: drop odhcpd leases with wrong field count
eb5ac25380e7 postgresql: security bump to 9.5.14
43f14b81120a unbound: limit outside script source to init funciton scope
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index f6c3287..6adccea 100644
--- a/modules
+++ b/modules
@@ -6,7 +6,7 @@ LEDE_COMMIT=184fe11483b9595f31a028356f8cc18fb18536e3
 
 PACKAGES_OPENWRT_REPO=https://github.com/openwrt/packages.git
 PACKAGES_OPENWRT_BRANCH=lede-17.01
-PACKAGES_OPENWRT_COMMIT=338690b2f79e2c7090be4e9adbb19b452c9e3c36
+PACKAGES_OPENWRT_COMMIT=40da7ecf21ffe1f3523ffa430c406e1db58ce3d4
 
 PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
 PACKAGES_GLUON_COMMIT=be2c35785994e443d895225c7240474a46f64f5e
-- 
2.5.0

